spec/models/room_spec.rb

```diff
@@ -3,5 +3,7 @@
 RSpec.describe Room, type: :model do
   describe "relationships" do
     it { should belong_to :hotel }
+    it { should have_many :guests }
+    it { should have_many :guest_rooms }
```

Comment:
Nice work testing at the unit and feature level! If you were to encapsulate the get_guest_count model method as I mentioned above, you would want to add an instance method test for it here.